Adding language about polling vs. webhook triggers for Github. (#11097)
PipedreamHQ/pipedream: components/github/README.md

# Getting Started

## Github Triggers: Webhooks vs. Polling

> The Github triggers are built in way that configures the trigger as a webhook if you are the admin of a repo, and falls back to polling if you are not the admin of a repo.

Reviewer: ```suggestion
The Github triggers in Pipedream enable you to get notified immediately via a webhook if you have `admin` rights on the repo you're watching. Otherwise you can still poll for updates at a regular interval for any other repo where you might not have `admin` rights.
```

Reviewer: Doesn't need to be this language exactly, but I think yours could use a little more context for users who might not be familiar with Pipedream.

> **Example: New or Updated Issue (Admin)**

Reviewer: What does (Admin) mean here? That's a little confusing to me

Author: Dropped this part, and just explained with the two examples.

> The user is an administrator of the repo, and this trigger will be configured as a webhook - so any time there is a new or updated issue in the repo, an event will automatically be emitted.

Reviewer: ```suggestion
If you are an admin on the repo, this trigger will be configured as a webhook — so any time there is a new or updated issue in the repo, an event will immediately get emitted.
```

<img src="https://res.cloudinary.com/dpenc2lit/image/upload/v1710954167/Screenshot_2024-03-20_at_9.58.01_AM_s1yych.png" width=600> 

> **Example: New or Updated Issue (Non-Admin)**

Reviewer: The `(Admin)` vs `(Non-Admin)` terminology is a bit confusing to me, as an uninformed user, especially since we're talking about repo-specific access, and not user-specific.

Author: Makes sense - dropped this.

> The user is not an administator of the repo, and has an option to the cadence by which to poll for new or updated issues.

Reviewer: ```suggestion
If you do not have `admin` rights on the repo you're watching, you can configure the Pipedream trigger to poll for updates on a regular interval.
```

Reviewer: Again, does not need this exact language, but I think your wording could be improved a little (and was maybe missing a word?).
